### PR TITLE
Issue 76 schema template

### PR DESCRIPTION
**Pull request by turbomam**
#76 

@cmungall I wrote `test_schema_metadata` and added 

```python
elif t == T_SCHEMA and isinstance(element, SchemaDefinition)
```
to `schemasheets/schema_exporter.py`, following your https://github.com/linkml/schemasheets/pull/77

but I think the exporting steps may have to be different, because we won't be iterating over a multiple schema rows, like we would for slots, classes or prefixes.

### CODE REVIEW

File: tests/input/schema_metadata.tsv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+schema	title
+> schema	prefix_reference
```
Comment (cmungall) [inline tests/input/schema_metadata.tsv:2]: typo?

File: tests/test_schema_exporter.py [hunk 2]
```diff
@@ -215,6 +250,3 @@ def test_export_metamodel_slots():
     # of this will change
     examples = s['examples']
     assert 'bibo:draft' == examples
-
-
-
```
Comment (cmungall) [inline tests/test_schema_exporter.py:220]: we need to start with tox for this repo!

File: tests/test_schema_exporter.py [hunk 3]
```diff
@@ -172,6 +174,39 @@ def test_enums():
     _roundtrip(schema, ENUM_SPEC)
 
 
+def test_prefixes():
```
[the remark this answers is not in the document]
Comment (turbomam) [reply tests/test_schema_exporter.py:177]: yeah, I treid to build upon your prefixes template fix. does that screw something up?

File: tests/test_schema_exporter.py (hunk 2 above)
Comment (turbomam) [reply tests/test_schema_exporter.py:220]: Ok, I watched a video about tox last night, but I don't think I'm ready to make that change yet.

**Pull request closed without merging**